Pipfile

```diff
@@ -22,6 +22,7 @@ uamqp = "*"
 requests = {extras = ["security"], version = "<2.32"}
 psycopg2 = "*"
 resolvelib = "==0.8.1"
+urllib3 = "*"
```

Comment:
I have urllib3 in my lock file without having specified it in the Pipfile.  I guess there is no danger to this though.

Reply:
If it's in our lockfile already then we don't need to put it here, especially as a wildcard. Either it should be a specific version (ideally everything should be but let's start here) or not added.